docs/source/users/options/minimizer_option.rst

```diff
@@ -144,7 +151,9 @@ The `gradient_free` minimizers are set as follows:
    See :ref:`extra_dependencies`.
 
 .. note::
-   BayesianOptimizer, TreeStructuredParzenEstimators and DecisionTreeOptimizer may be slow running	 
+   BayesianOptimizer, TreeStructuredParzenEstimators and DecisionTreeOptimizer may be slow running	and
```

Comment:
It looks like there's a tab here, looks like it was already lurking at the end of the line before this update.
```suggestion
   BayesianOptimizer, TreeStructuredParzenEstimators and DecisionTreeOptimizer may be slow running and
```